actions/build-push-docker/action.yml

```diff
@@ -228,6 +228,28 @@ runs:
           }}
         registries: ${{ inputs.aws-account-number }}
 
+    # To avoid running out of disk space (or no space left on device errors)
+    - name: Show disk space (before)
+      shell: bash
+      run: |
+        echo "Disk space (df -h):"
+        df -h
+        echo " "
+        echo "Disk space (df -i):"
+        df -i
+
+    - name: Free up disk space
+      uses: smartcontractkit/.github/actions/free-disk-space@free-disk-space/v1
+
+    - name: Show disk space (after)
+      shell: bash
+      run: |
+        echo "Disk space (df -h):"
+        df -h
+        echo " "
+        echo "Disk space (df -i):"
+        df -i
```

Comment:
This doesn't need to be done in this action. If the only use-case is for `ctf-build-image`, then call it in that action.